integration/PhpSpec/Console/Prompter/QuestionTest.php

```diff
@@ -3,12 +3,13 @@
 namespace integration\PhpSpec\Console\Prompter;
 
 use PhpSpec\Console\Prompter\Question;
+use PHPUnit\Framework\TestCase;
```

Comment:
When the forward compatibility changes were first introduced in PHPUnit, the FQCN was `phpunit\framework\TestCase`, which will cause autoload issues, it was updated in either 5.5 or 5.6 to use the case you're using here, so this part shouldn't be changed until phpspec 4.0 unless the minimum version of PHPUnit is increased.